### DISCUSSION TITLE
Grouping updates for different package managers

### DISCUSSION THREAD

**Discussion by bkonicek** [renovatebot/renovate#26236] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

I have Renovate set up to group updates into a single PR for a particular directory in my repo. This directory contains a kustomization file that deploys a Github release versioned CRD manifest, as well as the Helm chart to go along with it.

The updates are successfully grouped when there is a new release, however, the Helm chart's version is being updated to the Github release's semver value (in the most recent example, the Chart version is being updated from `1.13.2` to `v1.13.3` - notice the `v`)

Based on the logs it looks like it's correctly identifying each of the dependency types as `helm` and `github-release`. Is there a way to configure my group to distinguish between the Github release and the Helm chart in one group and update the values appropriately?

My config file looks like this - If there's a better way to configure the group I'm open to it:
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:recommended"
  ],
  "assignees": [
    "@bkonicek",
  ],
  "customManagers": [
    {
      "customType": "regex",
      "fileMatch": "kustomization.yaml",
      "matchStrings": ["https:\/\/github\\.com\/(?<depName>.*\/.*?)\/releases\/download\/(?<currentValue>.*?)\/"],
      "datasourceTemplate": "github-releases", 
    },
    {
      "customType": "regex",
      "fileMatch": "kustomization.yaml",
      "matchStrings": ["https:\/\/raw.githubusercontent.com\/(?<depName>[^\/]*\/[^\/]*)\/(?<currentValue>.*?)\/"],
      "datasourceTemplate": "github-tags",
    }
  ],
  "packageRules": [
    {
      "matchPackagePatterns": ["tailscale"],
      "matchManagers": ["regex"],
      "enabled": false
    },
    {
      "groupName": "helm and crds",
      "groupSlug": "helmCRDs",
      "matchPackagePatterns": [
        "cert-manager",
      ],
    },
  ],
}
```

PR diff is:

```diff
  resources:
+   - https://github.com/cert-manager/cert-manager/releases/download/v1.13.3/cert-manager.crds.yaml
    - clusterissuer-http-staging.yaml
    - clusterissuer-http.yaml
  
  helmCharts:
    - name: cert-manager
      releaseName: cert-manager
      namespace: cert-manager
+     version: v1.13.3
      repo: https://charts.jetstack.io
      valuesFile: values.yaml
```

### Logs (if relevant)

Link to Gist: https://gist.github.com/bkonicek/85a3addb15fb9107dbce71dc43e07d19

**Comment by secustor** [comment, marked as answer]
If everything you want to group is in a single path, than you can use [`matchFileNames`](https://docs.renovatebot.com/configuration-options/#matchfilenames)

**Reply by bkonicek** [reply]
Thanks, seems to have helped!
